packages/components/addon/components/hds/tabs/tab.hbs

```diff
@@ -1,6 +1,9 @@
 {{! template-lint-disable require-context-role }}
 {{! template-lint-disable no-invalid-role }}
 
+{{! Temporary: fix linter errors with using FlightIcon and BadgeCount inside role="tab" element }}
+{{! template-lint-disable require-presentational-children }}
+
 <li class={{this.classNames}} role="presentation" ...attributes>
   <a
```

Comment:
I added an explicit height of 36px now which is what I'm seeing in the Figma file. Let me know if this still isn't correct.

https://hds-components-git-hds-606-tab-options-hashicorp.vercel.app/components/tabs#showcase

(@heatherlarsen could use your eyes when you have a chance)

Reply:
I pushed up changes to spacing too. Let me know if you want a different implementation. @didoo

Reply:
there is still something not wright, but I think is the focus ring position; I'll try to attach a screenshot

Reply:
@KristinLBradley I took a look and left a few minor comments at the Vercel link. Let me know if you need clarification on anything. Great job though! Awesome to see them coming together 🙂

Reply:
@KristinLBradley OK, I may have found out the difference between Figma and the code.

Here's an overlay between the component in Figma and the component rendered in a browser:
<img width="400" alt="screenshot_1838" src="https://user-images.githubusercontent.com/686239/193257108-8a674bf0-5c60-46d6-9b04-ead9f0dcb925.png">
as you can see, they overlap almost perfectly (of course the horizontal size of text is different, different rendering engines)

But when I toggle the "focused" state, you can see that the two focus rings are different (the "magenta" one is the Figma one, I've changed the color to see the difference)
<img width="400" alt="screenshot_1839" src="https://user-images.githubusercontent.com/686239/193257611-58dbf19a-185d-4f83-9075-2fbbd62e28b1.png">
From what I see the vertical position should be reduced by a couple of pixels (both top and bottom). I _suspect_ this will impact the position of the indicator, but I think we can tune it later when it's implemented properly (with animation too).

I am going to open a small PR to improve the showcase of the Tabs component and subcomponents, so it's easier to debug this things (and plus we can have visual regression testing on them using Percy).

Reply:
I adjusted the focus ring and indicator positions. Please open another comment if you still see issues.

Reply:
No, all good now. Perfect!